File: CinemaProject.Angular/src/app/AdminPages/Models/amenity.ts
class Amenity {
  constructor(
    public id: string,
    public name: string,
    public cost: number,
    public extraPaymentPercent: number
  ) {}
}

class AmenityView {
  constructor(
    public id: string,
    public name: string,
    public cost: string,
    public extraPaymentPercent: number,
    public sessionId?: string,
    public isChecked: boolean = false
  ) {}
}

export { Amenity, AmenityView };

File: CinemaProject.WebAPI/CinemaProject.BLL/Models/Amenity.cs
using System;

namespace CinemaProject.BLL.Models
{
    public class Amenity
    {
        public Guid Id { get; set; }
        public string Name { get; set; }
        public decimal Cost { get; set; }
        public int ExtraPaymentPercent { get; set; }
    }
}
